### PR TITLE
Add pylint and mypy check for docker runs. 

### PR DESCRIPTION
**Pull request by MrLYG**

### CODE REVIEW

**Comment by fostiropoulos** [comment]
@MrLYG is this PR still in progress? if so can we convert to Draft?

**Comment by MrLYG** [comment]
> @MrLYG is this PR still in progress? if so can we convert to Draft?

Sorry, I forgot to add a description.
This PR is finished. And It should be merged.
This PR is used to add pylint and mypy check for docker runs, on local machine.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM continuumio/miniconda3:4.10.3
+
+WORKDIR /usr/src/app
+
+LABEL maintainer="dev@iordanis.xyz"
```
Comment (fostiropoulos) [inline Dockerfile]: mail@iordanis.me

**Review by fostiropoulos — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (MrLYG) [reply Dockerfile]: ok

**Review by fostiropoulos — APPROVED** [review]

**Pull request merged**